### ISSUE TITLE
Error on first startup of web_1

### ISSUE THREAD

**Issue by neflu** [cyllab/CalcUS#1]
**Describe the bug**
When starting CalcUS, I receive a loop of errors when starting web_1. This list of errors persists over time and is actually growing.

**To Reproduce**
1. Install Prerequisites (python packages, C++ requirements)
2. Generate environment through generate_env.bat
3. Start via start.bat
4. Error occurs when web_1 attempts to initalize. Error loops over and over for 5+ min.

**Screenshots**
![CalcUS_Error](https://user-images.githubusercontent.com/94268524/146712472-3c778b59-4a65-437d-a031-401cbd1e4b79.png)

**Configuration**
- Whether you use the prebuild Docker container (default) or run CalcUS from source.
> Prebuilt
- Host operating system (Windows, Linux, OSX...)
> Windows 10, but I have WSL2 installed as well. All my dependencies, etc. are installed on Windows.
- If the bug is mainly visual (like overlapping elements, missing section of the page) or related to web plugins, please specify your Web browser.
> Firefox, but I can't get to the stage of opening calcUS.

**Additional context**
I've already tried re-running start.bat, but this does not fix the issue. All the necessary prerequisites (python and c++ redistributable) are already installed.

**Comment by RaphaelRobidas** [comment]
Thank you for your issue report. This is caused by the different end of line (EOL) standards between DOS-based systems (such as Windows) and Unix-based systems (such as the Linux distribution which CalcUS is running). The files in the repository should have the Unix EOL character, but modifying them on Windows can change the EOL. You can verify this for example by opening a file in Notepad and look at the bottom right corner; in one of the tiles, it should indicate either Unix (LF) or Windows (CRLF). This crash due to the EOL is of course problematic and I am working on a robust fix.

In the meantime, you can change the EOL character to the Unix one (LF) instead of the DOS ones (CRLF). Notepad does not appear to have this feature, but other text editors should (for example Notepad++). I have done some tests and I get the same error if I change the EOL of precisely `scripts/run_web.sh`. This is most likely the file that you will need to edit.

Let me know if the issue persists.

**Comment by neflu** [comment]
Thank you for the quick response! I modified `run_web.sh` in Notepad++ as per the [following](http://www.sql313.com/index.php/43-main-blogs/maincat-dba/62-using-notepad-to-change-end-of-line-characters)
1.    Click on Search > Replace (or Ctrl + H)
2.    Find what:  \r\n
3.    Replace with: \n
4.    Search Mode: select Extended
5.    Replace All

Unfortunately, I get the following error when I attempt to access CalcUS at localhost:8080:

`web_1          | CalcUS version based on commit 15ab551
web_1          | No changes detected
web_1          | CalcUS version based on commit 15ab551
web_1          | No changes detected in app 'frontend'
web_1          | CalcUS version based on commit 15ab551
web_1          | Operations to perform:
web_1          |   Apply all migrations: admin, auth, axes, contenttypes, frontend, sessions
web_1          | Running migrations:
web_1          |   No migrations to apply.
web_1          | CalcUS version based on commit 15ab551
web_1          | CalcUS version based on commit 15ab551
web_1          | Superuser account found, creation aborted
web_1          | [2021-12-20 17:59:27 +0000] [29] [INFO] Starting gunicorn 20.1.0
web_1          | [2021-12-20 17:59:27 +0000] [29] [INFO] Listening at: http://0.0.0.0:8000 (29)
web_1          | [2021-12-20 17:59:27 +0000] [29] [INFO] Using worker: sync
web_1          | [2021-12-20 17:59:27 +0000] [30] [INFO] Booting worker with pid: 30
web_1          | CalcUS version based on commit 15ab551
web_1          | 2021-12-20 12:59:37,290 [ERROR]  log: Internal Server Error: /
web_1          | Traceback (most recent call last):
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/defaulttags.py", line 1034, in find_library
web_1          |     return parser.libraries[name]
web_1          | KeyError: 'global_tags'
web_1          |
web_1          | During handling of the above exception, another exception occurred:
web_1          |
web_1          | Traceback (most recent call last):
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/exception.py", line 47, in inner
web_1          |     response = get_response(request)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/core/handlers/base.py", line 181, in _get_response
web_1          |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
web_1          |   File "/calcus/frontend/views.py", line 128, in home
web_1          |     return render(request, 'frontend/home.html')
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/shortcuts.py", line 19, in render
web_1          |     content = loader.render_to_string(template_name, context, request, using=using)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loader.py", line 62, in render_to_string
web_1          |     return template.render(context, request)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/backends/django.py", line 61, in render
web_1          |     return self.template.render(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 170, in render
web_1          |     return self._render(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 162, in _render
web_1          |     return self.nodelist.render(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 938, in render
web_1          |     bit = node.render_annotated(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 905, in render_annotated
web_1          |     return self.render(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 127, in render
web_1          |     compiled_parent = self.get_parent(context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 124, in get_parent
web_1          |     return self.find_template(parent, context)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loader_tags.py", line 103, in find_template
web_1          |     template, origin = context.template.engine.find_template(
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/engine.py", line 125, in find_template
web_1          |     template = loader.get_template(name, skip=skip)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loaders/cached.py", line 58, in get_template
web_1          |     template = super().get_template(template_name, skip)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/loaders/base.py", line 29, in get_template
web_1          |     return Template(
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 155, in __init__
web_1          |     self.nodelist = self.compile_nodelist()
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 193, in compile_nodelist
web_1          |     return parser.parse()
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 478, in parse
web_1          |     raise self.error(token, e)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/base.py", line 476, in parse
web_1          |     compiled_result = compile_func(self, token)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/defaulttags.py", line 1091, in load
web_1          |     lib = find_library(parser, name)
web_1          |   File "/usr/local/lib/python3.9/site-packages/django/template/defaulttags.py", line 1036, in find_library
web_1          |     raise TemplateSyntaxError(
web_1          | django.template.exceptions.TemplateSyntaxError: 'global_tags' is not a registered tag library. Must be one of:
web_1          | admin_list
web_1          | admin_modify
web_1          | admin_urls
web_1          | cache
web_1          | details_tags
web_1          | folder_tags
web_1          | i18n
web_1          | l10n
web_1          | log
web_1          | static
web_1          | tz`

**Comment by RaphaelRobidas** [comment]
Yes, this is my bad, I forgot to include a file into the repository earlier. In any case, the version that I just pushed (commit f3f6d22) should fix both the original issue and this one. I tested it on Windows 10 as well and it appears to work.

You might notice that the footer says "Version unknown" - that's fine for now, I'm still working through this issue. It will not change anything in the functionalities.
